tests/mockserver.py

```diff
@@ -17,6 +18,12 @@
 from scrapy.utils.python import to_bytes, to_unicode
 
 
+def get_ephemeral_port():
+    s = socket.socket()
+    s.bind(("", 0))
```

Comment:
Hm, so you said there was an issue because Splash tests are binding port for 0.0.0.0 (the same as ""?), and it was problematic? Should we try finding a port by binding to localhost only?